src/utils/airtable.ts

```diff
@@ -80,6 +80,7 @@ export async function prepare_signup_data_for_airtable(
       Schulform: data.schoolTypes.value?.[0],
       Geburtsdatum: data.birthYear.value + `-01-01`,
       Kontaktperson: data.nameContact.value,
+      Kontaktpersonen: data.nameContact?.value,
```

Comment:
is this just added to prevent the airtable error?

Reply:
There are actually two fields in the chapter pages. One is just getting the name and another of is automatically linking it to the Kontaktpersonen Table. We should decide which way is better at some point in the future.